src/app/shared/versions.ts

```diff
@@ -68,7 +69,10 @@ export abstract class Versions extends EntryTab {
   getIconClass(columnName): IconDefinition {
     return this.dockstoreService.getIconClass(columnName, this.sortColumn, this.sortReverse);
   }
-  convertSorting(): string {
+  convertSorting(mode): string {
+    if (mode && mode === DockstoreTool.ModeEnum.HOSTED) {
+      this.sortColumn = 'id';
+    }
     return this.sortReverse ? '-' + this.sortColumn : this.sortColumn;
```

Comment:
What happens, and does it matter, when mode is not set? I guess you wouldn't have the versions without workflow being set on line 99 without the workflow...

Also, technically this method could return undefined, so since you're here you should probably update the method signature. And/or set sortColumn to null if the condition on line 73 is not true.

Reply:
In the case mode is not set, it should default it to last_modified/built, which was what this function had previously always set it to. I think that is a sane default since it works with most all non-hosted versions.